src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -36,29 +36,26 @@
 */
 public class ADDR_Contact_TDTM extends TDTM_Runnable {
     
-    /* @description Flag to prevent recursive call on insert */
-    public static Boolean alreadyRunAfterInsert = false;
-    /* @description Flag to prevent recursive call on update */
-    public static Boolean alreadyRunAfterUpdate = false;
     /* @description Flag used to indicate if this class is running as a result of an automatic Account creation caused by a Contact insert/update */
     public static Boolean afterAutomaticAccInsert = false;
```

Comment:
There *might* be a problem with this static variable but I cannot create a scenario where it fails. it gets set to true in ACCT_IndividualAccounts_TDTM...if it gets set to true in the first set of 200 records, it'll be true immediately for all remaining sets of 200. There might be a very special case vulnerability but hard to say